rmgpy/kinetics/surface.pyx

```diff
@@ -426,8 +512,67 @@ cdef class SurfaceArrhenius(Arrhenius):
         A helper function used when pickling a SurfaceArrhenius object.
         """
         return (SurfaceArrhenius, (self.A, self.n, self.Ea, self.T0, self.Tmin, self.Tmax, self.Pmin, self.Pmax,
-                                   self.uncertainty, self.comment))
+                                   self.coverage_dependence, self.uncertainty, self.comment))
+
+    cpdef double get_rate_coefficient(self, double T, double P=0.0) except -1:
+        """
+        Return the rate coefficient in the appropriate combination of m^3,
+        mol, and s at temperature `T` in K.
+        """
+        cdef double A, n, Ea, T0
+        A = self._A.value_si
+        n = self._n.value_si
+        Ea = self._Ea.value_si
+        T0 = self._T0.value_si
 
+        return A * (T / T0) ** n * exp(-Ea / (constants.R * T))
+
+    def to_cantera_kinetics(self):
```

Comment:
I suggest this cantera stuff is moved to a separate pull request, addressing https://github.com/ReactionMechanismGenerator/RMG-Py/issues/2117
(if this is indeed needed, and distinct from the parent Arrhenius class)